packages/components/src/Badge/Badge.component.js

```diff
@@ -116,7 +114,7 @@ Badge.propTypes = {
 	t: PropTypes.func.isRequired,
 	style: PropTypes.object, // eslint-disable-line react/forbid-prop-types
 	className: PropTypes.string,
-	display: PropTypes.oneOf([Badge.SIZES.small, Badge.SIZES.large]),
+	display: PropTypes.oneOf([SIZES.small, SIZES.large]),
```

Comment:
```suggestion
	display: PropTypes.oneOf(values(SIZES.small)),
```
with a 
```
import { values } from 'lodash';
```
Just a suggestion, to make it more evolutive if a badge size is remove / add

Reply:
I would prefer Object.values in that case